lib/calendar.js

```diff
@@ -74,17 +75,23 @@ var Calendar = _react2.default.createClass({
     this.props.onClickOutside(event);
   },
   getDateInView: function getDateInView() {
-    var _props = this.props;
-    var selected = _props.selected;
-    var openToDate = _props.openToDate;
+    var _props = this.props,
+        selected = _props.selected,
+        openToDate = _props.openToDate;
 
     var minDate = (0, _date_utils.getEffectiveMinDate)(this.props);
     var maxDate = (0, _date_utils.getEffectiveMaxDate)(this.props);
     var current = (0, _moment2.default)();
     if (selected) {
       return selected;
+    } else if (minDate && maxDate && openToDate && openToDate.isBetween(minDate, maxDate)) {
```

Comment:
Yeah, this (and the other comments) also looks to me like I just didn't update the lib after merging the changes into our fork :(